index.md

```diff
@@ -128,7 +117,7 @@ ms.assetid:
                                         <div class="card">
                                             <div class="cardImageOuter">
                                                 <div class="cardImage">
-                                                    <img src="./docs/images/hub/net-gs-3.svg" alt="" />
+                                                    <img src="articles/images/hub/logo_NETframework.svg" alt="" />
```

Comment:
the warnings are probably coming from these changed links. can you keep the links in the format "./docs/images/hub/" so we have clean builds?